src/main/java/frc/robot/subsystems/TankSubsystem.java

```diff
@@ -0,0 +1,52 @@
+
+package frc.robot.subsystems;
+
+import static frc.robot.Constants.TankConstants.*;
+
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.motorcontrol.MotorUtil;
+
+public class TankSubsystem extends SubsystemBase {
+    private final WPI_TalonSRX leftmotor = MotorUtil.createTalonSRX(MOTOR_BACKLEFT);
+    private final WPI_TalonSRX leftmotor2 = MotorUtil.createTalonSRX(MOTOR_FRONTLEFT);
+    private final WPI_TalonSRX rightmotor = MotorUtil.createTalonSRX(MOTOR_BACKRIGHT);
+    private final WPI_TalonSRX rightmotor2 = MotorUtil.createTalonSRX(MOTOR_FRONTRIGHT);
+
+    public double forwardpower = 0;
+    public double turnpower = 0;
+    public double scaler;
+
+    public TankSubsystem() {
+
+
+        leftmotor.setNeutralMode(NeutralMode.Brake);
+        leftmotor2.setNeutralMode(NeutralMode.Brake);
+        rightmotor.setNeutralMode(NeutralMode.Brake);
+        rightmotor2.setNeutralMode(NeutralMode.Brake);
+
+
+        leftmotor2.follow(leftmotor);
+        rightmotor2.follow(rightmotor);
+
+        leftmotor.setInverted(true);
+        leftmotor2.setInverted(true);
```

Comment:
You can use `setInverted(InvertType)` as a cleaner way of making a follower have the same inversion as the main. See [last year's shop project code](https://github.com/grt192/2021ShopProj/blob/6thPerClawMech/src/main/java/frc/robot/subsystems/tank/TankSubsystem.java#L34) and the [CTRE docs](https://docs.ctre-phoenix.com/en/latest/ch13_MC.html#follower).